app/assets/stylesheets/pages/_show.scss

```diff
@@ -36,6 +36,7 @@
 .button-pink {
   background-color:  #EA526F;
   color: white;
+  width: 100%;
```

Comment:
Can you change this so that there is a separate class for 100% width as this might change other parts of the style